gensrc/proto/data.proto

```diff
@@ -38,6 +38,7 @@ package starrocks;
 option java_package = "com.starrocks.proto";
 
 import "types.proto";
+import "tablet_schema.proto";
```

Comment:
if IndexType is the only type to be referred, may consider move IndexType into types.proto?